connect/runtime/src/test/java/org/apache/kafka/connect/runtime/errors/RetryWithToleranceOperatorTest.java

```diff
@@ -250,97 +258,90 @@ public void execAndHandleRetriableError(long errorRetryTimeout, int numRetriable
             assertTrue(retryWithToleranceOperator.failed());
         }
 
-        EasyMock.verify(mockOperation);
-        PowerMock.verifyAll();
+        verifyNoMoreInteractions(exitLatch);
+        verify(mockOperation, times(successExpected ? numRetriableExceptionsThrown + 1 : numRetriableExceptionsThrown)).call();
     }
 
     @Test
     public void testExecAndHandleNonRetriableError() throws Exception {
         MockTime time = new MockTime(0, 0, 0);
-        CountDownLatch exitLatch = PowerMock.createStrictMock(CountDownLatch.class);
+        CountDownLatch exitLatch = mock(CountDownLatch.class);
         RetryWithToleranceOperator retryWithToleranceOperator = new RetryWithToleranceOperator(6000, ERRORS_RETRY_MAX_DELAY_DEFAULT, ALL, time, new ProcessingContext(), exitLatch);
         retryWithToleranceOperator.metrics(errorHandlingMetrics);
 
-        EasyMock.expect(mockOperation.call()).andThrow(new Exception("Test")).times(1);
-
-        // expect no call to exitLatch.await() which is only called during the retry backoff
-
-        replay(mockOperation, exitLatch);
+        when(mockOperation.call()).thenThrow(new Exception("Test"));
 
         String result = retryWithToleranceOperator.execAndHandleError(mockOperation, Exception.class);
         assertTrue(retryWithToleranceOperator.failed());
         assertNull(result);
 
-        EasyMock.verify(mockOperation);
-        PowerMock.verifyAll();
+        // expect no call to exitLatch.await() which is only called during the retry backoff
+        verify(mockOperation, times(1)).call();
```

Comment:
Is there any useful difference between this assertion and one without `times(1)`? Otherwise, we can simplify (and add some consistency with line 314 below):
```suggestion
        verify(mockOperation).call();
```

Reply:
No difference, just thought it could be more explicit here. I've changed it to be consistent 👍